test/trento_web/controllers/v1/suse_manager_controller_test.exs

```diff
@@ -92,4 +95,48 @@ defmodule TrentoWeb.V1.SUSEManagerControllerTest do
       |> assert_schema("NotFound", api_spec)
     end
   end
+
+  describe "retrieve upgradable packages related patches" do
+    test "should return relevant patches grouped by package id", %{conn: conn, api_spec: api_spec} do
+      insert_software_updates_settings()
+
+      expect(Trento.SoftwareUpdates.Discovery.Mock, :get_patches_for_package, 3, fn _ ->
+        {:ok, build_list(10, :patch_for_package)}
+      end)
+
+      %PatchesForPackagesResponse{
+        patches: [
+          %PatchesForPackage{package_id: _, patches: _},
+          %PatchesForPackage{package_id: _, patches: _}
+        ]
+      } =
+        conn
+        |> get(
+          "/api/v1/software_updates/packages?package_ids[]=#{Faker.UUID.v4()}&package_ids[]=#{Faker.UUID.v4()}"
```

Comment:
Just curious, is it Phoenix that makes the URL parameters for a list specified like this?

As opposed to something like `"/api/v1/software_updates/packages?package_ids=#{package_1},#{package_2}"`

Reply:
I tried something like that, but it didn't work and Phoenix was seeing all the payload as a string; I tried to use `qs` in our JavaScript setup as well and a serialized array was resulting in a string of this form (with square brackets).

Having a look at the specifications it's pretty much framework specific, but on the other hand this form looks like the one that established as a "soft" standard :+1: